### PR TITLE
Authentication

### LINKED ISSUE #4: Add email verification
When new user registers, we need to verify their email and identity.

### PR DESCRIPTION
**Pull request by Naveen-Jayanna**
Added the email verification and validation feature

### CODE REVIEW

File: src/Backend/app.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 from flask import Flask, request, jsonify
 from flask_cors import CORS
 import json
-from src.Backend.utils import *
+from utils import *
```
Comment (samarth-p) [inline src/Backend/app.py]: Please revert it to the previous version

File: src/Backend/utils.py [hunk 2]
```diff
@@ -5,13 +5,15 @@
...
-        host=constants["host"], user=constants["user"], password=constants["password"], database=constants["database"])
-except:
-    pass
+from dbconfig import constants
+import smtplib, ssl
+
+# try:
+connection = mysql.connector.connect(
+    host=constants["host"], user=constants["user"], password=constants["password"], database=constants["database"])
+print("Hello World")
```
Comment (samarth-p) [inline src/Backend/utils.py]: Please revert this as was earlier

File: src/Backend/utils.py [hunk 3]
```diff
@@ -5,13 +5,15 @@
...
 import mysql.connector
 from ast import literal_eval as make_tuple
-from src.Backend.dbconfig import constants
-
-try:
-    connection = mysql.connector.connect(
-        host=constants["host"], user=constants["user"], password=constants["password"], database=constants["database"])
-except:
-    pass
+from dbconfig import constants
```
Comment (samarth-p) [inline src/Backend/utils.py]: This one as well

File: src/Backend/app.py (hunk 1 above)
Comment (Naveen-Jayanna) [reply src/Backend/app.py]: Reverted

File: src/Backend/utils.py (hunk 2 above)
Comment (Naveen-Jayanna) [reply src/Backend/utils.py]: Removed

File: src/Backend/utils.py (hunk 3 above)
Comment (Naveen-Jayanna) [reply src/Backend/utils.py]: Reverted

**Review by crmgogo — APPROVED** [review]
Looks good to me

**Pull request merged**